src/cascadia/TerminalSettingsModel/GlobalAppSettings.cpp

```diff
@@ -37,7 +37,7 @@ static constexpr std::string_view ConfirmCloseAllKey{ "confirmCloseAllTabs" };
 static constexpr std::string_view SnapToGridOnResizeKey{ "snapToGridOnResize" };
 static constexpr std::string_view EnableStartupTaskKey{ "startOnUserLogin" };
 static constexpr std::string_view AlwaysOnTopKey{ "alwaysOnTop" };
-static constexpr std::string_view UseTabSwitcherKey{ "useTabSwitcher" };
+static constexpr std::string_view TabSwitcherModeKey{ "useTabSwitcher" };

```

Comment:
```suggestion
static constexpr std::string_view TabSwitcherModeKey{ "tabSwitcherMode" };
```
Curious. It'd be a bit more work, but why not rename the setting to `tabSwitcherMode` in the JSON. And add legacy parsing for `useTabSwitcher`?

Reply:
... This isn't a bad idea. I'll play with it in the morning - might make the parsing a lot trickier, or I could just leave `tabSwitcherMode` accepting a `true|false|"inOrder"|"mru"` enum, and then just parse _both_ keys into the same member...

Otherwise I'll have to parse `useTabSwitcher` into a bool, then convert that to the right enum value, then try parsing the `tabSwitcherMode` enum

Reply:
I'm ok with parsing _both_ keys into the same member. It's a bit annoying, but I think it makes more sense as a name for the setting too (and future-proofs it).